Dockerfile

```diff
@@ -3,12 +3,12 @@
 #
 FROM golang:1.15 AS builder
 
-ARG RESTIC_VERSION=0.10.0
-ARG RESTIC_SHA256=067fbc0cf0eee4afdc361e12bd03b266e80e85a726647e53709854ec142dd94e
+ARG RESTIC_VERSION=0.11.0
+ARG RESTIC_SHA256=f559e774c91f1201ffddba74d5758dec8342ad2b50a3bcd735ccb0c88839045c
```

Comment:
Looks like you've taken the checksum of the amd64 binary, but we are building from source so this fails. I will fix it.

Reply:
My bad, sorry!